src/ElectricPokemon.java

```diff
@@ -0,0 +1,82 @@
+import java.util.Arrays;
+import java.util.List;
+
+public class ElectricPokemon extends Pokemon {
+
+    private final String food;
+    private final String sound;
+
```

Comment:
Het gebruik van subklasse heeft alleen bestaansrecht als de subklassen variëren van elkaar.  Als elke subklasse deze variabele krijgt kan je deze beter definiëren in de superklasse